src/main/java/roomescape/repository/ReservationRepository.java

```diff
@@ -0,0 +1,14 @@
+package roomescape.repository;
+
+import java.util.List;
+import java.util.Optional;
+
+import roomescape.domain.Reservation;
+
+public interface ReservationRepository {
+
+    Optional<Reservation> findById(Long id);
```

Comment:
```suggestion
    Optional<Reservation> findById(Long id);

    default Reservation getById(Long id) {
        return findById(id)
            .orElseThrow(() -> Exception.withDetail("id: " + id));
    }
```
저희 동아리에서는 이런식으로 사용하더라구요

Reply:
참고해보겠습니다 감사합니다